instructions/ms-sql-dba.instructions.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
This hasn't been added to the readme. Ensure you run the `update-readme.js` script before submitting.